### ISSUE TITLE
Drop support for Internet Explorer 8

### ISSUE THREAD

**Issue by philwareham** [textpattern/textpattern#555]
Currently in 4.6dev we support IE8 (this means additional stylesheets - both public and admin side) plus SVG fallbacks to PNG and adherence to v1.x of jQuery. Also Selectivizr can then be removed from vendors directory.

We need to plan to end-of-life the support for IE8 in future releases to ease future development. Probably post-4.6 release? Maybe sooner?

**Comment by vanmelick** [comment]
Microsoft drops support for IE8 on january 12th 2016.
Finding out what market share IE8 has is difficult:
0.5% http://www.w3schools.com/browsers/browsers_explorer.asp
16% http://venturebeat.com/2015/05/01/chrome-passes-25-market-share-ie-and-firefox-slip/
I suspect it's around 1-2%. Is that low enough to drop support next year?

**Comment by philwareham** [comment]
We are hoping for a 4.6 release in January. Wonder if I can drop it for 4.6 or wait for 4.7. It makes extra work for the Classic and Remora themes as I need to rewrite those a lot for 4.6 (if I have time, otherwise just Hive goes in for now).

**Comment by petecooper** [comment]
@vanmelick said: "I suspect it's around 1-2%."

StatCounter agrees: [http://gs.statcounter.com/#browser_version-ww-monthly-201410-201509](http://gs.statcounter.com/#browser_version-ww-monthly-201410-201509)

**Comment by phiw13** [comment]
On the other hand, Netmarketshare pegs IE 8 market share at 11.7 %: 
http://www.netmarketshare.com/browser-market-share.aspx?qprid=2&qpcustomd=0

The question boils down to this: will you, for Textpattern 4.6, do lots of JS stuff that could benefit from Jquery 2.x (or require it)? See my comment in [the forums](http://forum.textpattern.com/viewtopic.php?pid=296045#p296045).

**Comment by philwareham** [comment]
IE8 support has now been stripped in the admin side:

https://github.com/textpattern/textpattern/commit/fbbb788ce5c92c529cdf769affe0404ae1abca95

The public theme has also been done bit I haven't imported it into the core as yet.

**Comment by Bloke** [comment]
Great stuff. Do we need to code something in `_to_4.6.0.php` to delete removed files like selectivizr from people's installations, or is that too invasive?

**Comment by philwareham** [comment]
I would leave the files in there during upgrade, just in case they are used by other stuff. They won't do any harm, just clutter.

**Comment by philwareham** [comment]
Once admin-layout-update is merged, this issue can be closed.

**Comment by philwareham** [comment]
Done, I actually removed this earlier than planned in master because we potentially plan to amend the whole `vendor` directory soon.
